src/contributor-guide/dod/backward-compatibility.md

```diff
@@ -0,0 +1,36 @@
+---
+group: dod
+title: DoD - Backward Compatibility
+---
+
+## Overview
+
+All backward incompatible changes should be approved and covered by static tests to meet the "Definition of Done" (DoD).
+
+## Backward Compatibility
+
+### Functional Backward Compatibility
+
+**Functional backward compatibility** means the behaviour of the application is preserved.
+
+Existing product features and functionality must be retained during any changes to the code.
+Any backward-incompatible functional changes must be approved by a product owner.
+The documentation should explain the justification and provide the "business value".
+
+### Technical Backward Compatibility
+
+**Technical backward compatibility** means the technical interfaces are preserved.
+
+Technical interfaces include PHP interfaces or classes, CLI commands, URLs or any other interfaces that can be used by 3rd-party developer, system integrator, or user of Magento.
```

Comment:
should we also mention MFTF?

Reply:
I did not change the content of DoD apart from TCO analysis and static tests coverage for BIC topics, just decomposition it into several pages for better readability. Mentioning MFTF can be proposed in a separate PR.